pkg/test/store/postgres/test_container.go

```diff
@@ -40,17 +41,22 @@ func resourceDir() string {
 
 func (v *PostgresContainer) Start() error {
 	ctx := context.Background()
+
 	buildArgs := map[string]*string{}
+
 	mode := "standard"
 	if v.WithTLS {
 		mode = "tls"
 	}
-	uniqueId := strconv.Itoa(rand.Int())
 	buildArgs["MODE"] = &mode
+
 	// Add a uniqueId to make each image different, this was causing flakiness as test-container
 	// deletes the image that it builds unconditionally during teardown and fails if the image doesn't exist.
 	// In the case of parallel tests it's possible that the same image was used in multiple tests and the second teardown would fail.
+	r := rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
Maybe use: `GinkgoRandomSeed()` (This seed is in the log, makes it reproduceable still).